src/main/java/ch/njol/skript/expressions/ExprDurability.java

```diff
@@ -36,78 +36,80 @@
 import ch.njol.skript.util.slot.Slot;
 import ch.njol.util.coll.CollectionUtils;
 
-/**
- * @author Peter Güttinger
- */
-@Name("Data/Damage Value")
-@Description({"The data/damage value of an item/block. Data values of blocks are only supported on 1.12.2 and below.",
-		"You usually don't need this expression as you can check and set items with aliases easily, ",
-		"but this expression can e.g. be used to \"add 1 to data of &lt;item&gt;\", e.g. for cycling through all wool colours."})
-@Examples({"set damage value of player's tool to 10",
-		"set data value of target block of player to 3",
-		"add 1 to the data value of the clicked block",
-		"reset data value of block at player"})
-@Since("1.2")
+@Name("Damage Value/Durability")
+@Description("The damage value/durability of an item.")
+@Examples({
+	"set damage value of player's tool to 10",
+	"reset the durability of {_item}",
+	"set durability of player's held item to 0"
+})
+@Since("1.2, INSERT VERSION (durability reversed)")
 public class ExprDurability extends SimplePropertyExpression<Object, Long> {
 
+	private boolean durability;
+
 	static {
-		register(ExprDurability.class, Long.class, "((data|damage)[s] [value[s]]|durabilit(y|ies))", "itemtypes/blocks/slots");
+		register(ExprDurability.class, Long.class, "(damage[s] [value[s]]|durability:durabilit(y|ies))", "itemtypes/slots");
 	}
-	
+
+	@Override
+	public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelayed, ParseResult parseResult) {
+		durability = parseResult.hasTag("durability");
+		return super.init(exprs, matchedPattern, isDelayed, parseResult);
+	}
+
 	@Override
 	@Nullable
-	public Long convert(final Object o) {
-		if (o instanceof Slot) {
-			final ItemStack i = ((Slot) o).getItem();
-			return i == null ? null : (long) ItemUtils.getDamage(i);
-		} else if (o instanceof ItemType) {
-			ItemStack item = ((ItemType) o).getRandom();
-			return (long) ItemUtils.getDamage(item);
+	public Long convert(Object object) {
+		ItemStack itemStack = null;
+		if (object instanceof Slot) {
+			itemStack = ((Slot) object).getItem();
+		} else if (object instanceof ItemType) {
+			itemStack = ((ItemType) object).getRandom();
 		}
-		return null;
+		if (itemStack == null)
+			return null;
+		long damage = ItemUtils.getDamage(itemStack);
+		return durability ? itemStack.getType().getMaxDurability() - damage : damage;
 	}
-	
+
 	@Override
 	@Nullable
-	public Class<?>[] acceptChange(final ChangeMode mode) {
-		switch (mode) {
-			case ADD:
-			case SET:
-			case RESET:
-			case REMOVE:
-			case DELETE:
-				return CollectionUtils.array(Number.class);
-		}
-		return null;
+	public Class<?>[] acceptChange(ChangeMode mode) {
```

Comment:
I know Lime said to change it, but this becomes problematic if we were to ever add another mode, so I think the switch is better.